### ISSUE TITLE
Add color to either question or response to increase readability

### ISSUE THREAD

**Issue by jin-ds** [raiyanyahya/prompt#4]
**Is your feature request related to a problem? Please describe.**
It is hard to read what the AI's response is. Using a color should help with this.

**Describe the solution you'd like**
Either make the response or the question more visible by introducing a color/

**Describe alternatives you've considered**
Use ASCII-like art to highlight the response.

Thanks for considering it!

Note: I love having `prompt` in my command-line tool. I'm using Hyper on macOS—and it's a great way to quickly access OpenAI.

**Comment by raiyanyahya** [comment]
@jin-ds Thank you for the feature request! I think its a good idea and I will add it in my next release.

**Comment by jin-ds** [comment]
Lovely! I'm not too familiar with GitHub, but I believe when this is implemented and this FR is closed, I will get notified, right?

I also tried locating an update command to update the application, but wasn't able to find anything.

For a noob like myself, would you be able to add this in the readme doc (on how to update promptcli)? Thanks so much!

**Comment by raiyanyahya** [comment]
I will add a comment to this issue and you will be notified :+1: 

I will also add an "update" section to the cli. If you want to do it right now run "pip3 install -U promptcli".

**Comment by jalotra** [comment]
hey, this is WIP ? Would be a nice feature !

**Comment by raiyanyahya** [comment]
Yup I am testing this feature! Will let you know once this is done. Hopefully today.
